devops/charts/server/README.md

```diff
@@ -3,5 +3,5 @@ helm install vdr-proxy ./devops/charts/server -f ./devops/charts/server/values_d
 ```
 
 ```console
-helm template vdr-proxy ./devops/charts/server -f ./devops/charts/server values_dev.yaml | oc apply -n c2a2c4-dev -f -
+helm template vdr-proxy ./devops/charts/server -f ./devops/charts/server/values_dev.yaml | oc apply -n c2a2c4-dev -f -
```

Comment:
```suggestion
helm install vdr-proxy ./devops/charts/server -f ./devops/charts/server/values_dev.yaml
```

Reply:
If you need the namespace to change or be set you can put it in the values file or use `--set-string namespace=abc123-dev`

Reply:
done